ce/api/stats.py

```diff
@@ -91,13 +102,13 @@ def stats(sesh, id_, time, area, variable):
 
     try:
         df = sesh.query(DataFile).filter(DataFile.unique_id == id_).one()
-        fname = df.filename
+        resource = df.filename if not is_thredds else apply_thredds_root(df.filename)
```

Comment:
This is going to bite us if we ever want to make THREDDS calls from a CE server via a URL (maybe we don't?). Parameters supplied via URL are stringified, so if someone says `multistats?ensemble_name=p2a_rules&model=anusplin&variable=pr&emission=historical&time=0&timescale=seasonal&is_thredds=false` the value of `is_thredds` will be `"false"` instead of `False`. `"false"`, unlike `False` is truthy.